src/main/scala/org/allenai/deep_qa/experiments/datasets/Science.scala

```diff
@@ -227,7 +227,23 @@ object ScienceCorpora {
     ("elastic search index url" -> "aristo-es1.dev.ai2") ~
     ("elastic search index port" -> 9300) ~
     ("elastic search cluster name" -> "aristo-es") ~
-    ("elastic search index name" -> "busc")
+    ("elastic search index name" -> Seq("busc"))
+}
+
+object AristoDefaultCorpora {
+  def aristoDefaultElasticSearchIndex(numResults: Int): JValue =
```

Comment:
Instead of creating a new object, I would just put this method into the `ScienceCorpora` object.